### PR TITLE
Adds new available Azure locations

### PR DESCRIPTION
**Pull request by danielestevez**
Returned by Location API but not whitelisted yet
reference https://azure.microsoft.com/en-ca/regions/

### CODE REVIEW

File: azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java [hunk 1]
```diff
@@ -38,11 +38,17 @@
    US_GOV_VIRGINIA("US Gov Virginia", "US-VA"),
    NORTH_CENTRAL_US("North Central US", "US-IL"),
    SOUTH_CENTRAL_US("South Central US", "US-TX"),
+   WEST_CENTRAL_US("West Central US", ""),
```
Comment (nacx) [inline azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java]: US-WY

File: azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java [hunk 2]
```diff
@@ -38,11 +38,17 @@
...
    WEST_US("West US", "US-CA"),
+   WEST_US_2("West US 2", ""),
```
Comment (nacx) [inline azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java]: US-WA

File: azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java [hunk 3]
```diff
@@ -38,11 +38,17 @@
...
    NORTH_EUROPE("North Europe", "IE"),
+   UK_SOUTH("UK South", "GB-LND"),
+   UK_WEST("UK West", "GB-CRF"),
    WEST_EUROPE("West Europe", "NL"),
    EAST_ASIA("East Asia", "HK"),
    SOUTH_EAST_ASIA("Southeast Asia", "SG"),
+   KOREA_CENTRAL("Korea Central", "KR-11"),
+   KOREA_SOUTH("Korea South", ""),
```
Comment (nacx) [inline azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/Region.java]: KR-26

**Comment by danielestevez** [comment]
:up: Where did you get those locations info? https://azure.microsoft.com/en-ca/regions/ doesn't show them

**Comment by nacx** [comment]
Pushed to [master](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/83c58388) and [2.0.x](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/624155af).

**Pull request closed without merging**